packages/lesswrong/components/posts/PostsItem2.jsx

```diff
@@ -145,6 +148,9 @@ const styles = (theme) => ({
   startTime: {
     width: START_TIME_WIDTH,
     justifyContent: "center",
+    fontWeight: 300,
+    fontSize: "1rem !important",
```

Comment:
We can leave this in, but only if you promise to refactor it before it goes live so that it doesn't need the `!important`